### DISCUSSION TITLE
Getting " ⏎ " this symbol as an output on controller when editor is blank

### DISCUSSION THREAD

**Discussion by vishnureji** [singerdmx/flutter-quill#845] [Mobile] [answered]
when I was printing the controller's output, I was getting the" ⏎ "  symbol when the editor was blank.

`print(_summaryNoteController.document.toDelta().toString());`

how to remove this.? I need to check whether the controller is blank or not.

**Comment by dungngminh** [comment, marked as answer]
You can check the document is empty or not, using method `isEmpty` of `Document`.
Example:  
``` dart
 if (_quillController.document.isEmpty()) {
      // foo function bla..bla...
   return;
 }
```
